### ISSUE TITLE
NPE in KeyValueClient.getValuesAsString()

### ISSUE THREAD

**Issue by lbertrand** [OrbitzWorldwide/consul-client#194]
If this method is called with a non existing key, a NPE is thrown, instead of the method returning an empty list

Simple unit test
```java
public class SimpleTest {
    @Test
    public void testNPE() {
        Consul consul = Consul.newClient();
        List<String> shouldBeEmpty = consul.keyValueClient().getValuesAsString("unknownKey");
        assertTrue(shouldBeEmpty.isEmpty());     
    }
}
```

**Comment by rickfast** [comment]
Thanks for providing a unit test. I'll see if I can get to this next week.

**Comment by rickfast** [comment]
fixed by ec0f9acbfa9494201722a24e2ac06743bc6591ed
